handbook/onboarding/design.md

```diff
@@ -27,12 +33,21 @@ Make sure that you are **watching** all relevant repositories so that you don't
 
 Make sure all designers are given "editor" access to the [shared Launch Pad Design subfolder](https://drive.google.com/drive/u/0/folders/1Zfe25r3D77hGdyMkj0tlxHNa-r7fAq1d), which is owned by the `team@ubclaunchpad.com` account.
 
+Also, if your team decides to use google drive, make sure you are added to that by your project lead. Also make sure that you are linking your gdrive assets to your Github repository for easy accessibility.
+
 ### Figma
 
-TODO
+Make sure that you have a Figma account which you can create here: [Figma](https://www.figma.com/). It's completely free to use, web based and its the best collaborative design tool out there!
+* You can also download the Figma app on your desktop here: [Download Link](https://www.figma.com/downloads/)
+
+Other alternatives available out there are Adobe XD and Sketch. If you plan to use any of those tools, it is recommended that you import your designs into Figma so that you can share your prototypes and wireframe files with your developers.
 
 ## Workflows
 
+The Agile Method is the workflow that teams in LaunchPad usually follow. It is important that you familiarize youself with this type of workflow especially working in cross-functional product teams. 
```

Comment:
```suggestion
The Agile Method is the workflow that teams in Launch Pad usually follow. It is important that you familiarize youself with this type of workflow especially working in cross-functional product teams - you can learn more about it in our [handbooks' sprint planning guide](../project-management/sprints.md).
```

😄

Reply:
Oh you got me there, forgot to add in the link!! Nice!! haha